scripts/benchmark/run_dense_stress_observation_envelope.py

```
@@ -0,0 +1,671 @@
+
+def _load_fixture(path: pathlib.Path) -> dict[str, Any]:
+    """Load the dense-pedestrian-stress trace fixture."""
+    with open(path, encoding="utf-8") as fh:
+        return json.load(fh)
+
+
+def _git_head() -> str:
+    """Return the short git HEAD, or empty string on failure."""
+    try:
+        result = subprocess.run(
+            ["git", "rev-parse", "--short", "HEAD"],
```

Ruff (0.15.15)

[error] Starting a process with a partial executable path

(S607)